SOP/SOP-push.adoc

```diff
@@ -321,43 +334,44 @@ Init Containers:
     State:          Terminated
       Reason:       Completed
       Exit Code:    0
-      Started:      Fri, 12 Jul 2019 08:51:28 -0300
-      Finished:     Fri, 12 Jul 2019 08:51:53 -0300
+      Started:      Thu, 26 Sep 2019 12:09:52 +0300
+      Finished:     Thu, 26 Sep 2019 12:09:52 +0300
     Ready:          True
     Restart Count:  0
     Environment:
       POSTGRES_SERVICE_HOST:  example-unifiedpushserver-postgresql
     Mounts:
-      /var/run/secrets/kubernetes.io/serviceaccount from example-unifiedpushserver-token-5cmlp (ro)
+      /var/run/secrets/kubernetes.io/serviceaccount from example-unifiedpushserver-token-cszr4 (ro)
 Containers:
   ups:
-    Container ID:   docker://28fc71000c66d9223ab6e340a030491c4348a0b51979237de04488fe18282337
-    Image:          docker.io/aerogear/unifiedpush-wildfly-plain@sha256:62ecab1e74e3b1a7b2ef1d9eb7594f29bcf6b55702c269c9deebdadf8aea8a8a
-    Image ID:       docker-pullable://docker.io/aerogear/unifiedpush-wildfly-plain@sha256:62ecab1e74e3b1a7b2ef1d9eb7594f29bcf6b55702c269c9deebdadf8aea8a8a
+    Container ID:   docker://2cdc9c3703f274053e724b589ac24e1ba12db52a07158b68ffd7ddec328fcb51
+    Image:          quay.io/aerogear/unifiedpush-configurable-container:2.3
+    Image ID:       docker-pullable://quay.io/aerogear/unifiedpush-configurable-container@sha256:b5874b82bf4c7f13d63aa1ad806b2c306f8696fd21b7f8fa5c8697872af672ab
     Port:           8080/TCP
     Host Port:      0/TCP
     State:          Running
-      Started:      Fri, 12 Jul 2019 08:51:57 -0300
-    Last State:     Terminated
-      Reason:       Error
-      Exit Code:    255
-      Started:      Wed, 03 Jul 2019 21:45:20 -0300
-      Finished:     Fri, 12 Jul 2019 08:48:01 -0300
+      Started:      Thu, 26 Sep 2019 12:09:55 +0300
     Ready:          True
-    Restart Count:  1
-    Liveness:       http-get http://:8080/rest/applications delay=60s timeout=2s period=10s #success=1 #failure=3
-    Readiness:      http-get http://:8080/rest/applications delay=15s timeout=2s period=10s #success=1 #failure=3
+    Restart Count:  0
+    Limits:
+      cpu:     1
+      memory:  2Gi
+    Requests:
+      cpu:      500m
+      memory:   512Mi
+    Liveness:   http-get http://:8080/rest/applications delay=120s timeout=10s period=10s #success=1 #failure=3
+    Readiness:  http-get http://:8080/rest/applications delay=15s timeout=2s period=10s #success=1 #failure=3
     Environment:
-      POSTGRES_SERVICE_HOST:  example-unifiedpushserver-postgresql
+      POSTGRES_SERVICE_HOST:  <set to the key 'POSTGRES_HOST' in secret 'example-unifiedpushserver-postgresql'>  Optional: false
```

Comment:
I'm interested to know why we moved this to a secret?

Reply:
If I understand correctly, it was moved to the secret before this. Likely so that the deployment would be referencing the same information as any backup job.